modules/claims_api/spec/models/evidence_waiver_submission_spec.rb

```diff
@@ -6,24 +6,23 @@
   describe 'requiring fields' do
     context "when 'auth_headers' is not provided" do
       it 'fails validation' do
-        ews = ClaimsApi::EvidenceWaiverSubmission.new(encrypted_kms_key: 'bgdhjs')
+        ews = ClaimsApi::EvidenceWaiverSubmission.new
```

Comment:
Updated per @stiehlrod, as this explicit addition of this arg was causing a test to fail. This arg would be added by any way other than the kms_encrypted gem upon creating/saving/updating a record